docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+.NET Products
+=============
+
+.NET is a very flexible and naturally cross-platform [specification](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) for building developer products. It is used for all of the most popular app categories: desktop, mobile, cloud, gaming and IoT. 
+
+Major commercial and open source .NET products are listed below, in order of introduction.
```

Comment:
Probably true. I did one exercise, and I wasn't happy.

Alpha probably makes the most sense, as being the least controversial.